.github/workflows/cli-tests.yaml

```diff
@@ -242,7 +242,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest]
-        nix-version: [2.15.1, 2.16.1, 2.17.0, 2.18.0]
+        nix-version: [2.15.1, 2.16.1, 2.17.0, 2.18.0, 2.19.2]
```

Comment:
can we drop 2.16.1 and 2.18?

Reply:
I don't think there's a reason to drop older versions -- It's hard enough to upgrade Nix, I don't want to break early users